Dockerfile.cellguide_pipeline

```diff
@@ -0,0 +1,30 @@
+ARG BASE_TAG=branch-main
+
+FROM python:3.10
+RUN /usr/local/bin/python -m pip install --upgrade pip && \
+    apt update && apt -y install graphviz graphviz-dev && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip3 install awscli
+
+# For lighter weight Docker images
+ENV PIP_NO_CACHE_DIR=1
+
+WORKDIR /
+
+ADD requirements-cellguide-pipeline.txt requirements-cellguide-pipeline.txt
+RUN pip3 install -r requirements-cellguide-pipeline.txt
+
+ADD backend/__init__.py backend/__init__.py
+ADD backend/wmg/config.py backend/wmg/config.py
```

Comment:
probably not a big deal, but do we need the wmg files or were they accidentally copied over?

Reply:
to clarify: i mean this probably isn't a big deal to add the wmg files, but it is maybe a big deal if we forget to add cellguide specific files (that don't exist in this PR, but could exist in future ones)

Reply:
We need these for the future xD Not in this PR. Specifically, we need to be able to load the snapshot from WMG.